cityofaustin/atd-moped
# Use Data Grid in Staff List View

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/12334

## Testing
**URL to test:** 


Test locally, as this depends on the normalized workgroup


**Steps to test:**

Visit https://localhost:3000/moped/staff

Check that the add staff button works (but you cannot test adding a new user locally)
Same for the pencil/edit staff button, check that it loads, but dont test editing a user. 

I also fixed a small bug where the workgroup wasn't showing up on the edit staff page. 


---
#### Ship list
